src/features/Common/DeleteModal/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ModalDialog, ModalCloseButton } from '@edx/paragon';
+import { Button } from 'react-paragon-topaz';
+
+import 'features/Common/DeleteModal/index.scss';
+
+const DeleteModal = ({
+  isOpen, onClose, title, textModal, labelDeleteButton, handleDelete,
+}) => (
+  <ModalDialog
+    title={title || 'Delete'}
+    isOpen={isOpen}
+    onClose={onClose}
+    hasCloseButton
+    className="delete-modal"
+  >
+    <ModalDialog.Header className="py-4">
+      <ModalDialog.Title>{title || 'Delete'}</ModalDialog.Title>
```

Comment:
This title should go in the `propTypes`

```suggestion
      <ModalDialog.Title>{title}</ModalDialog.Title>
```